charts/hoprd-operator/values.yaml

```
@@ -155,6 +155,42 @@ operator:
+    enabled: false
+
+    ## @param operator.fastSync.schedule Schedule for Fast Sync
+    ##
+    schedule: "0 0 * * *"
+
+    source:
+
+      ## @param operator.fastSync.source.namespace Namespace of the Fast Sync source pod
+      ##
+      namespace: ""
+    
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)